Inter-views/Inter-views
# Connect login functionality front- and back-end

## **Types of Changes**


## **Purpose**
- connect front-end login component with back-end login server routes

## **Approach**
- utilize Redux Tool Kit to generate slices that manage actions, dispatch, and reducers

## **How to Test**
- npm run dev

enter credentials and click 'log in' button on dashboard // should generate a cookie (visible in dev tools) that is the same as the user's _id in the database

## **Files Altered**
```
src/App.tsx
src/Login.tsx
src/slices/authSlice.ts (formerly src/slices/signupSlice.ts)
src/store.ts
src/styles/styles.css
```
